data/json/items/gunmod/stock.json

```diff
@@ -123,5 +123,66 @@
     "dispersion_modifier": -1,
     "handling_modifier": 2,
     "min_skills": [ [ "weapon", 2 ] ]
+  },
+  {
+    "id": "butt_hook",
+    "type": "GUNMOD",
+    "name": "butt hook",
+    "description": "An attachment to the butt of the gun that is put under the armpit allowing for greater stability.",
+    "weight": "100 g",
+    "volume": "100 ml",
+    "price": 4750,
+    "price_postapoc": 50,
```

Comment:
Only 50 cents? I guess it should cost somewhat higher, at least several tens of dollars.

Reply:
Several tens of dollars seems insanely expensive for a single gunmod.
Here are all the items that are worth >$10 - a gunmod fits very poorly in this list.
Though there are things that should probably have their presence in their list considered, I don't think that provides reason for a gunmod to make that list.
<details>
<summary>Very long list</summary>
1200,AMMO,38super_fmj

1200,AMMO,cannon_shot

1200,AMMO,flintlock_ammo

1200,AMMO,flintlock_shot

1200,AMMO,reloaded_5x50dart

1200,COMESTIBLE,spaghetti_bolognese

1250,AMMO,plut_cell

1250,ARMOR,armguard_acidchitin

1250,ARMOR,armor_nomad_light

1250,ARMOR,back_holster

1250,ARMOR,backpack_tactical_large

1250,ARMOR,boots_chitin

1250,ARMOR,cleansuit

1250,ARMOR,duster_leather

1250,ARMOR,gauntlets_chitin

1250,ARMOR,grenadebandolier

1250,ARMOR,helmet_chitin

1250,ARMOR,hk_briefcase

1250,ARMOR,kevlar

1250,ARMOR,knee_high_boots

1250,ARMOR,mask_bal

1250,ARMOR,quiver_large_birchbark

1250,ARMOR,roller_shoes_off

1250,ARMOR,roller_shoes_on

1250,ARMOR,rucksack

1250,ENGINE,1cyl_combustion_large

1250,ENGINE,i4_combustion

1250,GENERIC,bat_metal

1250,GENERIC,bwirebat

1250,GENERIC,camera_control

1250,GENERIC,lajatang

1250,GENERIC,pike_inferior

1250,GENERIC,sail

1250,GENERIC,shillelagh

1250,GENERIC,spear_copper

1250,GUN,fn1910

1250,GUN,model_10_revolver

1250,GUN,pipe_combination_gun

1250,GUN,revolver_shotgun

1250,GUN,rugerlcp

1250,GUN,sw_619

1250,GUN,taurus_spectrum

1250,GUN,tokarev

1250,GUN,walther_ppk

1250,TOOL,fur_rollmat

1250,TOOL,knife_combat

1250,TOOL,knife_trench

1250,TOOL,kukri

1250,TOOL,lobotomizer

1250,TOOL,pickaxe

1250,TOOL,shock_epee

1250,TOOL,shock_foil

1250,TOOL,shock_sabre

1500,AMMO,300blk

1500,AMMO,380_p

1500,AMMO,bolt_cf

1500,AMMO,RPG-7_og7v

1500,ARMOR,armor_farmor

1500,ARMOR,armor_riot

1500,ARMOR,bandolier_pistol

1500,ARMOR,bandolier_rifle

1500,ARMOR,bandolier_shotgun

1500,ARMOR,boots_bone

1500,ARMOR,boots_bunker

1500,ARMOR,boots_combat

1500,ARMOR,bunker_coat

1500,ARMOR,coat_fur_sf

1500,ARMOR,gloves_tactical

1500,ARMOR,gloves_xlsurvivor

1500,ARMOR,hazmat_suit

1500,ARMOR,helmet_army

1500,ARMOR,hood_h20survivor

1500,ARMOR,jacket_leather_mod

1500,ARMOR,lsurvivor_pants

1500,ARMOR,molle_pack

1500,ARMOR,quiver_large

1500,ARMOR,rollerskates

1500,ARMOR,saddlebag

1500,ARMOR,sleeping_bag_roll

1500,ARMOR,tool_belt

1500,ARMOR,travelpack

1500,COMESTIBLE,adderall

1500,COMESTIBLE,caffeine

1500,COMESTIBLE,chaw

1500,COMESTIBLE,flu_shot

1500,COMESTIBLE,pie_maple

1500,COMESTIBLE,pizza_supreme

1500,COMESTIBLE,tobacco

1500,ENGINE,i6_diesel

1500,ENGINE,v6_combustion

1500,ENGINE,v6_diesel

1500,GENERIC,0

1500,GENERIC,drive_by_wire_controls

1500,GENERIC,i_staff

1500,GENERIC,motor

1500,GENERIC,motor_large

1500,GENERIC,mountable_cooler

1500,GENERIC,mountable_heater

1500,GENERIC,naginata_inferior

1500,GENERIC,reinforced_solar_panel

1500,GENERIC,vh_autoclave

1500,GENERIC,v_reaper_item_advanced

1500,GENERIC,xl_water_wheel

1500,GENERIC,xl_wind_turbine

1500,GUN,combination_gun

1500,GUN,minispeargun

1500,GUNMOD,gun_crossbow

1500,GUNMOD,riv_suppressor

1500,GUNMOD,suppressor

1500,GUNMOD,suppressor_compact

1500,GUN,mp18

1500,GUN,mp40

1500,GUN,shotgun_s

1500,GUN,sig_40

1500,GUN,smg_40

1500,GUN,smg_45

1500,GUN,speargun

1500,GUN,sten

1500,MAGAZINE,heavy_atomic_battery_cell

1500,MAGAZINE,storage_battery

1500,PET_ARMOR,rubber_armor_horse

1500,TOOL_ARMOR,mask_fsurvivorxl

1500,TOOL_ARMOR,mask_h20survivorxl

1500,TOOL_ARMOR,mask_lsurvivor

1500,TOOL_ARMOR,mask_survivorxl

1500,TOOL_ARMOR,mask_wsurvivorxl

1500,TOOL_ARMOR,solarpack

1500,TOOL_ARMOR,solarpack_on

1500,TOOL_ARMOR,wearable_atomic_light

1500,TOOL,blade_trap

1500,TOOL,bot_riotbot

1500,TOOL,bot_science_bot

1500,TOOL,char_smoker

1500,TOOL,dynamite_bomb

1500,TOOL,fire_ax

1500,TOOL,fishing_rod_professional

1500,TOOL,grenade_inc

1500,TOOL,halligan

1500,TOOL,large_repairkit

1500,TOOL,large_tent_kit

1500,TOOL,machete

1500,TOOL,press_dowel

1500,TOOL,rx12_injector

1500,TOOL,small_space_heater

1500,TOOL,tanto

1500,TOOL,tazer

1500,TOOL,UPS_off

1500,TOOL,water_mill

1500,TOOL,water_purifier

1500,TOOL,wind_mill

1500,WHEEL,hd_steel_drum

1600,AMMO,32_acp

1600,AMMO,46mm

1600,AMMO,57mm

1600,AMMO,shot_dragon

1600,AMMO,shot_he

1750,ARMOR,armor_nomad

1750,ARMOR,boots_acidchitin

1750,ARMOR,boots_larmor

1750,ARMOR,gauntlets_acidchitin

1750,ARMOR,helmet_acidchitin

1750,COMESTIBLE,cig

1750,ENGINE,v8_combustion

1750,ENGINE,v8_diesel

1750,GENERIC,nailbat

1750,GENERIC,shillelagh_weighted

1750,GENERIC,v_planter_item_advanced

1750,GUN,bigun

1750,GUN,colt_ro635

1750,GUN,glock_22

1750,GUN,hptcf380

1750,GUN,kp3at

1750,GUN,pistol_flintlock

1750,GUN,triple_launcher_simple

1750,PET_ARMOR,leather_harness_dog

1750,TOOL_ARMOR,mask_fsurvivor

1750,TOOL_ARMOR,mask_survivor

1750,TOOL_ARMOR,mask_wsurvivor

1800,AMMO,5x50dart

1800,ARMOR,garnet_gold_ring

1800,TOOL,machete_gimmick

1900,PET_ARMOR,leatherbone_harness_dog

2000,AMMO,700nx

2000,AMMO,arrow_exploding

2000,AMMO,chem_compositionb

2000,AMMO,gunpowder

2000,AMMO,lgpistol_primer

2000,AMMO,lgrifle_primer

2000,AMMO,RPG-7_ammo

2000,AMMO,shotgun_primer

2000,ARMOR,ammo_satchel

2000,ARMOR,armor_scavenger

2000,ARMOR,baldric_holster

2000,ARMOR,chainmail_feet

2000,ARMOR,chainmail_legs

2000,ARMOR,chainmail_vest

2000,ARMOR,chestpouch

2000,ARMOR,diamond_silver_earring

2000,ARMOR,dragonskinempty

2000,ARMOR,gloves_h20survivor

2000,ARMOR,gloves_lsurvivor

2000,ARMOR,gloves_survivor

2000,ARMOR,gloves_survivor_fingerless

2000,ARMOR,helmet_larmor

2000,ARMOR,modularvest

2000,ARMOR,pants_survivor

2000,ARMOR,sleeping_bag_fur

2000,ARMOR,sleeping_bag_fur_roll

2000,ARMOR,stillsuit

2000,ARMOR,swat_armor

2000,ARMOR,tacvest

2000,COMESTIBLE,pie_meat

2000,COMESTIBLE,pie_veggy

2000,COMESTIBLE,pur_tablets

2000,COMESTIBLE,vaccine_shot

2000,COMESTIBLE,vitamins

2000,ENGINE,aero_engine_light

2000,GENERIC,0

2000,GENERIC,cantilever_medium

2000,GENERIC,software_hacking

2000,GUN,90two40

2000,GUN,af2011a1_38super

2000,GUN,browning_a5

2000,GUN,bullet_crossbow

2000,GUN,doublespeargun

2000,GUN,fs2000

2000,GUN,hi_power_40

2000,GUN,hptjcp

2000,GUN,hptjhp

2000,GUN,kpf9

2000,GUN,m1911a1_38super

2000,GUN,makarov

2000,GUN,marlin_9a

2000,GUNMOD,shot_suppressor

2000,GUN,recurbow

2000,GUN,reflexbow

2000,GUN,shotgun_410

2000,GUN,shotgun_d

2000,GUN,skorpion_82

2000,GUN,walther_ccp

2000,GUN,walther_ppq_40

2000,GUN,woodgreatbow

2000,PET_ARMOR,chitin_harness_dog

2000,TOOL,anvil

2000,TOOL_ARMOR,mask_h20survivor

2000,TOOL_ARMOR,mask_hsurvivor

2000,TOOL,autoclave

2000,TOOL,battleaxe_inferior

2000,TOOL,bot_nursebot

2000,TOOL,bot_turret_riot

2000,TOOL,c4

2000,TOOL,char_purifier

2000,TOOL,cs_lajatang_off

2000,TOOL,dehydrator

2000,TOOL,ecs_lajatang_off

2000,TOOL,folding_bicycle

2000,TOOL,press

2000,TOOL,puller

2000,TOOL,RTG_coffee

2000,TOOL,vac_sealer

2000,TOOL,vortex_stone

2250,ARMOR,survivor_pack

2250,ENGINE,v12_combustion

2250,ENGINE,v12_diesel

2250,GUN,bond_410

2250,GUN,carbine_flintlock

2250,GUN,glock_31

2250,GUN,hi_power_9mm

2250,GUN,h&k416a5

2250,GUN,hptc9

2250,GUN,ksg

2250,GUN,lever_shotgun

2250,GUN,m320

2250,GUN,mossberg_500

2250,GUN,mossberg_500_security

2250,GUN,mossberg_930

2250,GUN,p220_10

2250,GUN,p226_357sig

2250,GUN,p320_357sig

2250,GUN,ppsh

2250,GUN,remington_870_breacher

2250,GUN,rifle_308

2250,GUN,rifle_flintlock

2250,GUN,rm120c

2250,GUN,sw_610

2250,GUN,walther_p38

2250,GUN,walther_ppq_45

2250,GUN,winchester_1887

2250,GUN,xd_10

2300,GUN,hk_mp5_10_semi

2300,GUN,TDI_10

2400,AMMO,5x50heavy

2400,GUN,mossberg_590

2500,AMMO,50bmg

2500,AMMO,bolt_explosive

2500,AMMO,chem_rdx

2500,AMMO,gunpowder_artillery

2500,AMMO,gunpowder_large_rifle

2500,AMMO,gunpowder_magnum_pistol

2500,AMMO,gunpowder_pistol

2500,AMMO,gunpowder_rifle

2500,AMMO,gunpowder_shotgun

2500,ARMOR,armor_lamellar

2500,ARMOR,armor_larmor

2500,ARMOR,bigback

2500,ARMOR,chainmail_arms

2500,ARMOR,chainmail_hands

2500,ARMOR,chainmail_hood

2500,ARMOR,entry_suit

2500,ARMOR,gloves_fsurvivor

2500,ARMOR,gloves_wsurvivor

2500,ARMOR,hood_xlsurvivor

2500,ARMOR,modularveststeel

2500,ARMOR,roller_blades

2500,ARMOR,survivor_belt_notools

2500,ARMOR,survivor_runner_pack

2500,ARMOR,torso_bandolier_shotgun

2500,COMESTIBLE,lsd

2500,COMESTIBLE,nic_gum

2500,COMESTIBLE,poppy_sleep

2500,COMESTIBLE,weed

2500,ENGINE,small_turbine_engine

2500,GENERIC,bokken

2500,GENERIC,can_sealer

2500,GENERIC,motor_enhanced

2500,GENERIC,programmable_autopilot

2500,GENERIC,scythe_war

2500,GENERIC,tearer

2500,GENERIC,turret_mount

2500,GENERIC,welding_components

2500,GUN,1895sbl

2500,GUN,90two

2500,GUN,american_180

2500,GUN,colt_lightning

2500,GUN,colt_saa

2500,GUN,cz75

2500,GUN,draco

2500,GUN,famas

2500,GUN,glock_17

2500,GUN,glock_18c

2500,GUN,glock_19

2500,GUN,glock_20

2500,GUN,glock_29

2500,GUN,glock_40

2500,GUN,hand_crossbow

2500,GUN,longrifle_flintlock

2500,GUN,m1911

2500,GUN,m79

2500,GUN,m9

2500,GUN,mac_10

2500,GUN,mac_11

2500,GUN,mauser_c96

2500,GUN,p08

2500,GUN,px4

2500,GUN,px4_40

2500,GUN,raging_bull

2500,GUN,remington_870

2500,GUN,skorpion_61

2500,GUN,usp_45

2500,GUN,usp_9mm

2500,GUN,walther_ppq_9mm

2500,GUN,winchester_1897

2500,GUN,witness_10

2500,MAGAZINE,large_storage_battery

2500,PET_ARMOR,acidchitin_harness_dog

2500,TOOL_ARMOR,goggles_ir

2500,TOOL,rapier

2500,TOOL,shocktonfa_off

2500,TOOL,survivor_machete

2500,TOOL,sword_xiphos

2500,TOOL,tool_small_primitive_fragmentation_device

2600,GUN,glock_21

2750,GUN,bfr

2750,GUN,carbine_flintlock_double

2750,GUN,deagle_44

2750,GUN,fn57

2750,GUN,m17

2750,GUN,m1911_10

2750,GUN,m1911_MEU

2750,GUN,remington700_270

2750,GUN,remington_870_express

2750,GUN,ruger_redhawk

2750,GUN,sharps

2750,GUN,surv_carbine_223

2750,GUN,sw629

2750,GUN,tec9

2750,GUN,tommygun

2750,TOOL,wakizashi

3000,AMMO,anesthetic

3000,AMMO,rebreather_filter

3000,AMMO,RPG-7_pg7vr

3000,ARMOR,armor_blarmor

3000,ARMOR,armor_chitin

3000,ARMOR,armor_cuirass

3000,ARMOR,armor_plarmor

3000,ARMOR,chainmail_hauberk

3000,ARMOR,chestrig

3000,ARMOR,gloves_hsurvivor

3000,ARMOR,gloves_plate

3000,ARMOR,helmet_lobster

3000,ARMOR,helmet_nasal

3000,ARMOR,hood_lsurvivor

3000,ARMOR,modularvestkevlar

3000,ARMOR,survivor_goggles

3000,ARMOR,survivor_rucksack

3000,COMESTIBLE,bacon_uncut

3000,COMESTIBLE,cocaine_topical

3000,COMESTIBLE,codeine

3000,COMESTIBLE,purifier_smart_shot

3000,COMESTIBLE,quikclot

3000,GENERIC,1st_aid

3000,GENERIC,2h_flail_steel

3000,GENERIC,atomic_coffeepot

3000,GENERIC,ifak

3000,GENERIC,solar_panel_v2

3000,GENERIC,sword_cane

3000,GENERIC,template_photonics

3000,GENERIC,turret_controls

3000,GUN,aksemi

3000,GUN,browning_blr

3000,GUN,colt_army

3000,GUN,colt_navy

3000,GUN,compgreatbow

3000,GUN,compositebow

3000,GUN,hk_mp7

3000,GUN,hk_ump45

3000,GUN,ksub2000

3000,GUN,longbow

3000,GUN,m11

3000,GUN,m38dmr

3000,GUN,mk23

3000,GUN,mosin44

3000,GUN,mosin91_30

3000,GUN,uzi

3000,PET_ARMOR,leather_armor_horse

3000,PET_ARMOR,leatherbone_armor_horse

3000,TOOL,adv_UPS_off

3000,TOOL,bot_c4_hack

3000,TOOL,bot_EMP_hack

3000,TOOL,bot_talon_m202a1

3000,TOOL,bot_turret

3000,TOOL,char_forge

3000,TOOL,combatsaw_off

3000,TOOL,e_combatsaw_off

3000,TOOL,EMPbomb

3000,TOOL,khopesh

3000,TOOL,l-stick

3000,TOOL,toolbox

3000,TOOL,toolbox_empty

3100,AMMO,9x18mmfmj

3250,ARMOR,survivor_duffel_bag

3250,GUN,m1903

3250,GUN,m1911-460

3250,GUN,pamd71z

3250,GUN,remington_1100

3250,GUN,win70

3250,TOOL,toolbox_workshop

3250,TOOL,toolbox_workshop_empty

3500,ARMOR,armor_acidchitin

3500,ARMOR,armor_lorica

3500,ARMOR,helmet_conical

3500,ARMOR,helmet_corinthian

3500,ARMOR,helmet_galea

3500,ARMOR,helmet_scavenger

3500,ARMOR,hood_wsurvivor

3500,ARMOR,modularvesthard

3500,ARMOR,pickelhaube

3500,GENERIC,ji

3500,GENERIC,motor_super

3500,GENERIC,qiang

3500,GENERIC,reinforced_solar_panel_v2

3500,GENERIC,spear_dory

3500,GUN,ar_pistol

3500,GUN,compbow

3500,GUN,crossbow

3500,GUN,fn_p90

3500,GUN,garand

3500,GUN,henry_big_boy

3500,GUN,hk_mp5

3500,GUN,hk_mp5k

3500,GUN,LAW

3500,GUN,m1014

3500,GUN,m1a

3500,GUN,m231pfw

3500,GUN,mosin44_ebr

3500,GUN,mosin91_30_ebr

3500,GUN,remington_700

3500,GUN,sig_mcx_rattler_sbr

3500,GUN,sks

3500,GUN,streetsweeper

3500,GUN,sw_500

3500,GUN,TDI

3500,PET_ARMOR,chainmail_harness_dog

3500,TOOL_ARMOR,goggles_nv

3500,TOOL,cutlass

3500,TOOL,dao

3500,TOOL,scimitar

3750,GUN,m2010

3750,GUN,ruger_mini

3750,GUN,sp2022

4000,AMMO,20x66_flare

4000,AMMO,20x66_flechette

4000,AMMO,20x66_shot

4000,AMMO,20x66_slug

4000,AMMO,RPG-7_tbg7v

4000,ARMOR,boots_xlsurvivor

4000,ARMOR,cuirass_lightplate

4000,ARMOR,helmet_barbute

4000,ARMOR,hood_fsurvivor

4000,ARMOR,hood_survivor

4000,ARMOR,legguard_lightplate

4000,ARMOR,lsurvivor_armor

4000,ARMOR,modularvestceramic

4000,ARMOR,survivor_vest

4000,ARMOR,xlsurvivor_suit

4000,COMESTIBLE,bfipowder

4000,GENERIC,atomic_lamp

4000,GENERIC,atomic_lamp_off

4000,GENERIC,atomic_light

4000,GENERIC,atomic_light_off

4000,GENERIC,minifreezer

4000,GUN,ak47

4000,GUN,ak74

4000,GUN,ar15

4000,GUN,bh_m89

4000,GUN,compositecrossbow

4000,GUN,cx4

4000,GUN,huge_crossbow

4000,GUN,ksg-25

4000,GUN,lemat_revolver

4000,GUN,m1918

4000,GUN,m202_flash

4000,GUN,pamd68

4000,TOOL,bot_secubot

4000,TOOL,cavalry_sabre

4000,TOOL,forge

4000,TOOL,grenade_emp

4000,TOOL,inflatable_boat

4000,TOOL,jian

4000,TOOL,knife_rm42

4000,TOOL,LAW_Packed

4000,TOOL,sword_bayonet

4000,TOOL,tool_improvised_barrel_bomb

4250,GUN,savage_111f

4500,ARMOR,sleeveless_duster_survivor

4500,COMESTIBLE,royal_beef

4500,GENERIC,generator_7500w

4500,GENERIC,spear_survivor

4500,GUN,an94

4500,GUN,arx160

4500,GUN,iwi_tavor_x95_300blk

4500,GUN,M24

4500,GUN,p226_9mm

4500,GUN,pseudo_m203

4500,GUN,saiga_12

4500,TOOL,arming_sword

4500,TOOL,katana

4500,TOOL,shock_staff

4750,GUN,SPAS_12

5000,AMMO,20x66_frag

5000,AMMO,20x66_inc

5000,AMMO,ampoule

5000,AMMO,atgm_heat

5000,AMMO,stimpack_ammo

5000,ARMOR,boots_lsurvivor

5000,ARMOR,depowered_helmet

5000,ARMOR,fancy_glasses_bifocal

5000,ARMOR,fancy_glasses_eye

5000,ARMOR,fancy_glasses_reading

5000,ARMOR,fancy_sunglasses_eye

5000,ARMOR,fancy_sunglasses_reading

5000,ARMOR,fancy_transition_glasses_eye

5000,ARMOR,fancy_transition_glasses_reading

5000,ARMOR,fsurvivor_suit

5000,ARMOR,h20survivor_suit

5000,ARMOR,helmet_kabuto

5000,ARMOR,helmet_plate

5000,ARMOR,lsurvivor_suit

5000,ARMOR,modularvestsuper

5000,ARMOR,shark_suit

5000,ARMOR,survivor_suit

5000,ARMOR,transition_glasses_eye

5000,ARMOR,transition_glasses_reading

5000,ARMOR,wsurvivor_suit

5000,ARMOR,xlhsurvivor_suit

5000,COMESTIBLE,aspirin

5000,COMESTIBLE,caff_gum

5000,COMESTIBLE,coke

5000,COMESTIBLE,crack

5000,COMESTIBLE,joint

5000,COMESTIBLE,meth

5000,COMESTIBLE,xanax

5000,ENGINE,medium_turbine_engine

5000,ENGINE,steam_watts_small

5000,GENERIC,catan

5000,GENERIC,estoc

5000,GENERIC,g_warhammer

5000,GENERIC,g_warhammer40k

5000,GENERIC,pike

5000,GENERIC,poleaxe

5000,GENERIC,RobofacCoin

5000,GUN,atgm_launcher

5000,GUN,briefcase_smg

5000,GUN,calico

5000,GUN,coilgun

5000,GUN,hybridbow

5000,GUN,m16a4

5000,GUN,m4a1

5000,GUN,mark19

5000,GUN,mgl

5000,GUNMOD,rm121aux

5000,GUN,trex_gun

5000,GUN,USAS_12

5000,MAGAZINE,huge_atomic_battery_cell

5000,PET_ARMOR,kevlar_armor_horse

5000,PET_ARMOR,superalloy_harness_dog

5000,TOOL,bot_rifleturret

5000,TOOL,broadsword

5000,TOOL,minireactor

5000,TOOL,scrambler

5200,TOOL,bullwhip_razor

5500,GENERIC,spear_steel

5500,GUN,acr_300blk

5500,GUN,ar10

5500,GUN,fn_fal

5500,GUN,oa93

5500,GUN,tavor_12

5500,GUN,v29

5500,GUN,weatherby_5

6000,AMMO,0

6000,ARMOR,aep_suit

6000,ARMOR,armor_plate

6000,ARMOR,boots_fsurvivor

6000,ARMOR,boots_h20survivor

6000,ARMOR,boots_plate

6000,ARMOR,boots_survivor

6000,ARMOR,boots_wsurvivor

6000,ARMOR,dragonskin

6000,ARMOR,shark_suit_faraday

6000,ARMOR,xlkevlar

6000,GENERIC,mace

6000,GENERIC,warhammer

6000,GUN,acr

6000,GUN,AT4

6000,GUN,bt_apc9k

6000,GUN,compcrossbow

6000,GUN,flamethrower

6000,GUN,hk417_13

6000,GUN,hk_g3

6000,GUN,hk_g36

6000,GUN,m110a1

6000,GUN,m2browning_sawn

6000,GUN,m3_carlgustav

6000,GUN,m4_cqbr

6000,GUN,rebar_rifle

6000,GUN,reflexrecurvebow

6000,GUN,rep_crossbow

6000,GUN,rm103a_pistol

6000,GUN,scar_l

6000,GUN,sig552

6000,GUN,steyr_aug

6000,PET_ARMOR,chitin_armor_horse

6000,TOOL,bot_crows_m240

6000,TOOL,butterfly_swords

6000,TOOL,longsword

6500,GUN,emp_gun

6500,GUN,m14ebr

6800,AMMO,762_m87

7000,ARMOR,hsurvivor_suit

7000,GUN,saiga_410

7500,AMMO,50_mk211

7500,ARMOR,fancy_sunglasses_bifocal

7500,ARMOR,fancy_transition_glasses_bifocal

7500,ARMOR,power_armor_helmet_basic

7500,ARMOR,transition_glasses_bifocal

7500,ENGINE,large_turbine_engine

7500,GUN,laser_rifle

7500,GUN,m240

7500,GUN,m60

8000,AMMO,12mm

8000,AMMO,20x66_exp

8000,AMMO,66mm_HEAT

8000,AMMO,8mm_caseless

8000,AMMO,m235tpa

8000,ARMOR,armor_samurai

8000,ARMOR,ballistic_vest_esapi

8000,ARMOR,boots_hsurvivor

8000,ARMOR,chainmail_suit

8000,ARMOR,duster_survivor

8000,ARMOR,trenchcoat_survivor

8000,COMESTIBLE,panacea

8000,ENGINE,steam_triple_small

8000,GENERIC,glaive

8000,GENERIC,morningstar

8000,GUN,gatling_mech_laser

8000,GUN,laser_cannon

8000,GUN,m249

8000,GUN,m4_carlgustav

8000,GUN,needlegun

8000,GUN,needlepistol

8000,GUN,plasma_rifle

8000,GUN,recon_mech_laser

8000,GUN,rm802

8000,PET_ARMOR,acidchitin_armor_horse

8000,TOOL_ARMOR,dimensional_anchor

8000,TOOL,bot_antimateriel

8000,TOOL,control_laptop

9000,ARMOR,anbc_suit

9000,ARMOR,chainmail_suit_faraday

9000,COMESTIBLE,contacts

9000,GUN,bfg50

9000,GUN,heavy_rail_rifle

9500,GENERIC,naginata

9500,TOOL,zweihander

10000,ARMOR,power_armor_frame

10000,ARMOR,power_armor_helmet_heavy

10000,ARMOR,power_armor_helmet_light

10000,COMESTIBLE,pills_sleep

10000,COMESTIBLE,prozac

10000,COMESTIBLE,thorazine

10000,ENGINE,steam_triple_medium

10000,GENERIC,halberd

10000,GENERIC,lucern_hammer

10000,GUN,m134

10000,GUN,m2browning

10000,GUN,plasma_gun

10000,GUN,rm11b_sniper_rifle

10000,GUN,rm20

10000,GUN,rm2000_smg

10000,GUN,rm228

10000,GUN,rm451_flamethrower

10000,PET_ARMOR,chainmail_armor_horse

10000,TOOL,anesthetic_kit

10000,TOOL_ARMOR,holo_cloak

10000,TOOL_ARMOR,rm13_armor

10000,TOOL,bot_dispatch

10000,TOOL,bot_mininuke_hack

10000,TOOL,bot_turret_speaker

10000,TOOL,mininuke

12000,AMMO,84x246mm_he

12000,AMMO,84x246mm_hedp

12000,AMMO,84x246mm_smoke

12000,ARMOR,armguard_lightplate

12000,ARMOR,armor_lightplate

12000,GUN,as50

12000,GUN,helsing

12000,GUN,hk_g80

12000,GUN,m107a1

12000,GUN,pneumatic_shotgun

12000,GUN,rm51_assault_rifle

12000,GUN,tac50

12000,GUN,tihar

12000,TOOL,battleaxe

12000,TOOL,bot_laserturret

12000,TOOL,nodachi

14000,GUN,hk_mp5sd

15000,ARMOR,depowered_armor

15000,COMESTIBLE,poppy_pain

15000,GUN,hm12

15000,GUN,rm298

15000,GUN,rm614_lmg

15000,TOOL_ARMOR,phase_immersion_suit

15000,TOOL,hoboreel

17500,GUN,rm88_battle_rifle

20000,AMMO,rocuronium

20000,COMESTIBLE,antifungal

20000,COMESTIBLE,antiparasitic

20000,COMESTIBLE,diazepam

20000,COMESTIBLE,heroin

20000,COMESTIBLE,iodine

20000,COMESTIBLE,prussian_blue

20000,COMESTIBLE,tramadol

20000,COMESTIBLE,weak_antibiotic

20000,TOOL,plastichoboreel

25000,COMESTIBLE,morphine

25000,TOOL_ARMOR,optical_cloak

25000,TOOL,bot_dispatch_military

30000,TOOL_ARMOR,power_armor_basic

40000,COMESTIBLE,antibiotics

40000,COMESTIBLE,oxycodone

40000,TOOL_ARMOR,power_armor_heavy

40000,TOOL_ARMOR,power_armor_light

50000,AMMO,8mm_hvp

50000,COMESTIBLE,prophylactic_antivenom

50000,GENERIC,mjolnir
60000,COMESTIBLE,strong_antibiotic
</details>

Reply:
It's the other items from the list might require balancing their costs. But that's off-topic.

In any case 50 cents seems insanely cheap for any thing, not to mention stuff that literally helps you to survive.

Reply:
> It's the other items from the list might require balancing their costs. But that's off-topic.

[Again](https://github.com/CleverRaven/Cataclysm-DDA/pulls?q=is%3Apr+is%3Amerged+price_postapoc+author%3Abrian-otten)? 
Do these not fit with [our pricing documentation](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/POSTAPOC_PRICE_GUIDE.md)?

I agree 50 cents is on the cheap side here, but tens of dollars is totally inconsisent.

Reply:
It costs 47,5$, I took that from an internet shop.
The post apo price is based on the scenario:
during zombie apocalypse go to the FEMA camp, full of refugees, with only a handful of scavengers and try to trade X for food, medicine, guns.